src/platforms/android/androidutils.cpp

```diff
@@ -218,4 +218,17 @@ QByteArray AndroidUtils::DeviceId() {
 void AndroidUtils::openNotificationSettings() {
   QAndroidJniObject::callStaticMethod<void>(
       "org/mozilla/firefox/vpn/qt/VPNUtils", "openNotificationSettings", "()V");
+}
+
+JNIEXPORT void JNICALL Java_org_mozilla_firefox_vpn_qt_VPNUtils_sendGleanEvent(
+    JNIEnv* env, jobject VPNUtils, jstring event) {
+  Q_UNUSED(VPNUtils);
+  const char* buffer = env->GetStringUTFChars(event, nullptr);
+  if (!buffer) {
+    return;
+  }
+  QString eventString(buffer);
+  logger.info() << "Glean Event via JNI:" << eventString;
+  emit MozillaVPN::instance()->recordGleanEvent(eventString);
```

Comment:
As you see here we are actaully recording the glean event. Sending a ping happens later and seperately. I think it would be better if all the android code also called it `recordGleanEvent` so that we don't get confused about the two concepts.